doc/nrf/app_dev/device_guides/nrf53/building_nrf53.rst

```diff
@@ -147,3 +147,133 @@ To program the nRF5340 DK from the command line, use either west (which uses nrf
 
             .. note::
                  The ``--verify`` command confirms that the writing operation has succeeded.
+
+.. _thingy53_building_pgming:
+
+Building and programming with Thingy:53
```

Comment:
This is already described in this page: https://ncsdoc.z6.web.core.windows.net/PR-20005/nrf/app_dev/device_guides/thingy53/building_thingy53.html.